streams/src/main/java/org/apache/kafka/streams/state/internals/WrappedStateStore.java

```diff
@@ -92,4 +94,12 @@ public void close() {
     public S wrapped() {
         return wrapped;
     }
+
+    protected static void captureException(final Runnable action, final List<RuntimeException> suppressed) {
```

Comment:
Should we have a "global utility" class that unifies all helper methods we use throughout the whole code base?

Reply:
Yeah, I was thinking that, but didn't mention it for fear of expanding the scope of this PR. We could use such a utility all over the place when we need to capture exceptions in closing.

I'll send a follow-up PR.